CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 - Add example with column rename macro ([PR](https://github.com/godatadriven/pytest-dbt-core/pull/19))
 
+## [0.1.1] - 2023-02-22
```

Comment:
You can remove this and add your entry on line two. The release script automatically adds the version number and date section header

Reply:
changed https://github.com/godatadriven/pytest-dbt-core/pull/25/commits/9c04c9242202d3573ee12a37a566f7a3bc60fcea